Solutions/CohesitySecurity/json_parser.sh

```diff
@@ -3,9 +3,9 @@ SCRIPT=$(realpath "$0")
 SCRIPTPATH=$(dirname "$SCRIPT")
 cd "$SCRIPTPATH"
 
-ClientId=$(cat ./cohesity.json | jq '."ClientId"' | sed 's/^"//g;s/"$//g')
-ClientKey=$(cat ./cohesity.json | jq '."ClientKey"' | sed 's/^"//g;s/"$//g')
-StartDaysAgo=$(cat ./cohesity.json | jq '."StartDaysAgo"' | sed 's/^"//g;s/"$//g')
+ClientId=$(cat ./cohesity.json | jq '."client_id"' | sed 's/^"//g;s/"$//g')
```

Comment:
Please change ClientId, ClientKey and StartDaysAgo to snake case as well.

Reply:
those are shell variable, should we need to follow the snake case style?

Reply:
We have to be consistent. The variables 'api_key', 'consumer_context' etc below use snake case.

Reply:
done